src/screens/login/LoginScreen.js

```diff
@@ -158,7 +164,13 @@ const LoginSteps = ({
   }
 
   const setInstanceData = useCallback(
-    async ({ instance, fqdn }) => {
+    /**
+     * Sets the instance data.
+     * @param {{ instance: Record<string, unknown>, fqdn: string }} instanceData - The first parameter object with `instance` as a record of string keys to unknown values, and `fqdn` as a string.
```

Comment:
instance is supposed to be a string, did you find a case where it was something else? 🤔

Reply:
updated